src/01-functions.js

```diff
@@ -74,7 +82,9 @@ function multiply() {}
  *  average(10, 6);
  *  //> 8 // 10 + 6 = 16; 16/2 = 8;
  */
-function average() {}
+function average(a, b) {
+  return (a+b)/2
```

Comment:
Jorge - first of all this is some great looking code!  I think here on like 86 you could space out the '/'  operator and the  Number 2 to make it slightly easier to read.